startup.sh

```diff
@@ -28,12 +30,10 @@ fi
 echo "Stopping frontend services..."
 docker-compose down
 
-if [[ -z "${AWS_ACCESS_KEY_ID:-}" || -z "${AWS_SECRET_ACCESS_KEY:-}" ]]; then
-  echo "!! AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY must be set in the environment." >&2
-  echo "!! Configure AWS access for either gds-di-development (sandpit) or di-auth-development (authdevs)." >&2
-  echo "!! For gds-cli: gds aws digital-identity-dev -- ${0}" >&2
-  exit 1
-fi
+export AWS_PROFILE="gds-di-development-admin"
```

Comment:
will this work for deploying to the authdevs too?

Reply:
currently no: I'm not certain what's actually being deployed: Is this not a 'run things locally' deal?

This was more of a backwards-compatible change. Can add in authdev stuff as well if needed?